src/components/elementView/powerView/powerView.tsx

```diff
@@ -295,7 +296,8 @@ export class PowerView extends React.PureComponent<
         {this.props.asset &&
         this.state.powerStatus &&
         this.state.statusLoaded &&
-        this.shouldShowPowerButtons() ? (
+        this.shouldShowPowerButtons() &&
+        !this.props.isMobile ? (
```

Comment:
so your plan is to show power status, but not allow for control of power from mobile, right?

Reply:
Right!